### PR TITLE
[datadog_ip_allowlist] add ip allowlist example

### PR DESCRIPTION
**Pull request by afrieds**
Add IP Allowlist example. Merge when approved.

### CODE REVIEW

**Review by drichards-87 — APPROVED** [review]

**Review by amaskara-dd — APPROVED** [review]

File: docs/resources/ip_allowlist.md [hunk 1]
```diff
@@ -10,7 +10,23 @@ description: |-
...
 
-
+## Example Usage
+
+```terraform
+resource "datadog_ip_allowlist" "example" {
+  enabled = false
+
+  entry {
+    cidr_block = "172.0.0.1/32"
```
Comment (srosenthal-dd) [inline docs/resources/ip_allowlist.md:20]: @afrieds this is a real public IP; can we switch to a private IP address which won't be in use?

One of the following would work great, and show that /32 isn't the only option possible. All are reserved for documentation, ref: https://en.wikipedia.org/wiki/Reserved_IP_addresses
* 192.0.2.0/24	
* 198.51.100.0/24
* 203.0.113.0/24

Comment (afrieds) [reply docs/resources/ip_allowlist.md:20]: Oops I meant to do 127. Yes of course. I will reopen. Sorry about that.

**Pull request merged**